css-sass/index.html

```diff
@@ -0,0 +1,321 @@
+            <div class="flex gap-2">
+              <div class=" w-1 h-1 flex items-center justify-center">
+                <img src="../assets/svg/check_item.svg"></img>
+              </div>
+              <p class="font-normal text-base leading-tight tracking-tight">Conectamos emprendimientos uruguayos con
+                organizaciones </p>
+            </div>
+            <div class="flex gap-2">
+              <div class=" w-1 h-1 flex items-center justify-center">
+                <img src="../assets/svg/check_item.svg"></img>
+              </div>
+              <p class="font-normal text-base leading-tight tracking-tight">Apoyamos emprendimientos enfocados en
+                productos inclusivos y sustentables
+              </p>
+            </div>
+            <div class="flex gap-2">
+              <div class=" w-1 h-1 flex items-center justify-center">
+                <img src="../assets/svg/check_item.svg"></img>
+              </div>
+              <p class="font-normal text-base leading-tight tracking-tight">Trabajamos en conjunto con empresas para
+                realizar acciones innovadoras y con un impacto positivo en sus equipos, en sus
+                clientes y en el ambiente.
+              </p>
+            </div>
+            <div>
+              <button type="button"
+                class="mt-1 py-0-9 px-4 rounded-lg border-none bg-primary-gradient text-white text-base tracking-tight leading-tight font-medium shadow-md cursor-pointer">SOLICITÁ
+                TU PRUEBA GRATIS
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="mt-5 w-80 m-inline flex flex-col">
+    <div class="flex justify-between items-center">
+      <h3 class="font-semibold text-2xl text-transparent-bgclip">Nuestros clientes</h3>
+      <div class="">
+        <img src="../assets/svg/vector_left.svg"></img>
+        <img src="../assets/svg/vector_right.svg" class="ml-3"></img>
+      </div>
+    </div>
+    <div class="flex gap-2">
+      <div class="flex flex-col gap-2 w-50">
+        <div class="mt-3 justify-between items-center">
+          <img src="../assets/svg/vector_quote.svg"></img>
+        </div>
+        <hr class="border-t-line">
+        <h6 class="mt-1 font-normal text-base tracking-ticht leading-tight">Gestiono todo rápidamente y me desentiendo
+          de la logística y la preocupación de si llegó o no el producto. Son un key importante en HR.
+        </h6>
+        <div class="mt-3 flex items-center gap-2">
+          <div class=" flex items-center rounded-xl shadow-xl bg-light-gray w-6 h-4">
+            <img src="../assets/images/logo_dlocal.jpeg" class="w-70 h-70 m-inline"></img>
+          </div>
+          <div>
+            <p class="text-base font-normal tracking-tight leading-tight">Florencia López</p>
+            <p class="text-base font-light tracking-tight leading-snug">Human Resources</p>
```

Comment:
Remember that some css properties are inherited, here you have a link in which you can check which properties are inherited in css: https://stackoverflow.com/questions/5612302/which-css-properties-are-inherited. There are a lot of places where we can take advantage of this fact in this file, but let's try it here with the text-base property by moving it to the wrapper div and that way avoid repetition, just to start thinking in this kind of things. This is something that we should keep in mind in the future to keep the code cleaner, specially if we are using the utility-classes approach since the markup and the styles coexist in the same place.

Reply:
Property (text-size: 1.6rem) applied to body